src/ds5/ds5-auto-calibration.cpp

```diff
@@ -1266,10 +1357,19 @@ namespace librealsense
                             {
                                 progress_callback->on_update_progress(static_cast<float>(20 + static_cast<int>(frame_counter * 60.0 / _total_frames)));
                             }
-                            _fill_factor[frame_counter + fw_host_offset] = calc_fill_rate(f);
+                            auto fr  = calc_fill_rate(f);
```

Comment:
Please change this param name to fill_rate

Reply:
Done